apps/heureka/src/components/components/ComponentsTopNavItem.jsx

```diff
@@ -4,7 +4,8 @@
  */
 
 import React from "react"
-import ComponentsListController from "./ComponentsListController"
+import ComponentsList from "./ComponentsList"
+import ListController from "../shared/ListController"
 // import Filters from "../filters/Filters"
 import { Messages, MessagesProvider } from "@cloudoperators/juno-messages-provider"
 
```

Comment:
Same thing with the naming as described under IssueMatches

Reply:
same here